test/AutoRemediateS3-002.test.js

```diff
@@ -0,0 +1,230 @@
+"use strict";
+
+const source = require('../functions/AutoRemediateS3-002');
+
+//const config = require('./config');
+const AWS = require("aws-sdk");
+
+const CCRuleCode = 'S3-002';
+const CCRuleName = 'BucketPublicReadAcpAccess';
+const allUsersURI = 'http://acs.amazonaws.com/groups/global/AllUsers';
+const readAcpPermission = "READ_ACP";
+const aclSkeleton = '{"Owner":"", "Grants":[]}'; // skeleton for new permission grants
+
+describe('S3-002 READ_ACP auto-remediation', () => {
+
+    describe('S3 ACL Grants shall transfer to new ACL except for READ_ACP grant', () => {
+
+        describe('S3 ACL with READ_ACP grant', () => {
+            it('is not transfered to new ACL', () => {
+                var aclNew = JSON.parse(aclSkeleton);
+                const readAcpGrant = JSON.parse('{ "Grantee": { "Type": "Group", "URI": "http://acs.amazonaws.com/groups/global/SomeUsers" }, "Permission": "READ_ACP" }');
```

Comment:
You don't need to use `JSON.parse`

```const readAcpGrant = JSON.parse('{ "Grantee": { "Type": "Group", "URI": "http://acs.amazonaws.com/groups/global/SomeUsers" }, "Permission": "READ_ACP" }');```

is equivalent to:
```
const readAcpGrant = { Grantee: { Type: "Group", URI: "http://acs.amazonaws.com/groups/global/SomeUsers" }, Permission: "READ_ACP" }
```

And it makes it easier to use variables etc.